tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -1001,6 +1001,62 @@ TEST(http_parser, fills_hdr_tbl_for_resp)
 	}
 }
 
+TEST(http_parser, cache_control_flags)
```

Comment:
New tests are only about parsing responses, requests are not checked, not all `TFW_HTTP_CC_*` flags are tested. No need to address this comment in the PR, since functional tests from tempesta-tech/tempesta-test#59 will cover the some cases. Just wanted to link this test case with #1059 issue.